integrations/crowdnode/src/main/res/drawable/ic_account_gradient.xml

```diff
@@ -1,40 +1,10 @@
-<!--
-  ~ Copyright 2022 Dash Core Group.
-  ~
-  ~ This program is free software: you can redistribute it and/or modify
-  ~ it under the terms of the GNU General Public License as published by
-  ~ the Free Software Foundation, either version 3 of the License, or
-  ~ (at your option) any later version.
-  ~
-  ~ This program is distributed in the hope that it will be useful,
-  ~ but WITHOUT ANY WARRANTY; without even the implied warranty of
-  ~ MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the
-  ~ GNU General Public License for more details.
-  ~
-  ~ You should have received a copy of the GNU General Public License
-  ~ along with this program.  If not, see <http://www.gnu.org/licenses/>.
-  -->
```

Comment:
The copyright doesn't need to be removed. We decided we should have it in the .xml files as well.
It's better if you add our copyright in the Android Studio -> Settings -> Copyright Profiles and choose it as default.